Sources/Puree/Output/BufferedOutput.swift

```diff
@@ -13,7 +13,7 @@ open class BufferedOutput: InstantiatableOutput {
         public let logs: Set<LogEntry>
         private(set) var retryCount: Int = 0
 
-        fileprivate init(logs: Set<LogEntry>) {
+        public init(logs: Set<LogEntry>) {
```

Comment:
Why need you change this?

Reply:
This initializer must be visible from subclasses of `BufferedOutput` because it is used in overridden `prepareChunk`.

However I think this is not a very good approach. The another option to avoid make this init public is to change return type of `prepareChunk` from `[Chunk]` to `[Set<LogEntry>]`.